experiment/build/generate_cloudbuild.py

```diff
@@ -45,33 +86,54 @@ def create_cloud_build_spec(buildable_images, docker_registry):
         if 'depends_on' in image:
             step['wait_for'] = []
             for dep in image['depends_on']:
-                step['wait_for'] += [dep]
-        cloud_build_spec['images'].append(name)
+                if 'base' in dep and not base:
+                    continue
+                step['wait_for'] += [_identity(dep)]
+            if len(step['wait_for']) == 0:
+                del step['wait_for']
+
+        cloud_build_spec['images'].append(
```

Comment:
If you have time, I'd like you try an experiment to see if we can optimize this.
Only `'${_REPO}/runners/${_FUZZER}/${_BENCHMARK}:${_EXPERIMENT}'` actually needs to be pushed. SO try doing that.
Another thing I'd like to check is that caching works properly in fuzzbench cloud project and inside.